src/components/implementation/no_interface/scheddev/fprr/sl_mod_fprr.c

```diff
@@ -35,6 +38,7 @@ void
 sl_mod_block(struct sl_thd_policy *t)
 {
 	ps_list_rem_d(t);
+	sl_timeout_mod_block(sl_mod_thd_get(t), 0);
```

Comment:
Is this just setting a timeout for this thread to "no timeout"?

This doesn't seem like the type of thing that we want in a scheduling module.  Shouldn't it be called for *every* scheduling module?  There will be one, sometime in the future, that forgets to call this...